nncf/tensorflow/graph/converter.py

```diff
@@ -87,24 +59,132 @@ def convert_keras_model_to_nncf_graph(model: tf.keras.Model) -> NNCFGraph:
     :param model: Keras model
     :return: NNCFGraph
     """
-    func_model = is_functional_model(model)
-    seq_model = is_sequential_model(model)
-
-    if not func_model and not seq_model:
-        RuntimeError('convert_keras_model_to_nncf_graph function supports '
-                     'only sequential or functional models')
+    converter = TFModelConverterFactory.create(model)
+    return converter.convert()
 
-    if func_model:
-        converter = FunctionalConverter(model)
-    else:
-        converter = SequentialConverter(model)
 
-    return converter.convert()
+class CustomLayerNodeInfo:
```

Comment:
Could you please add docstring for all classes and public methods?

Reply:
Done